README.md

```diff
@@ -95,10 +95,10 @@ Additional features -
 
 ## Why fastdup?
 
-- **Quality**: Find and remove anomalies and outliers from your dataset, including duplicates and similar images and videos at a large scale.
-- **Cost**: Reduce data operation costs by intelligently sampling high-quality or novel datasets before labeling and assessing labeled data quality.
-- **Scale**: fastdup's C++ graph engine is highly efficient and can handle up to 400M images on a single CPU machine.
-
+* **Fast**: Efficient C++ engine with speed up to 2 ms per image - process up to 8M images per hour with a 32-core machine.
+* **Scalable**: Handles up to 400M images on a single CPU machine.
+* **Unsupervised**: Runs on unlabeled (or labeled) image/video data.
+* **Cost**: Basic functions are free to use. Process up to 12M images on a $1 cloud machine budget. 
```

Comment:
I wouldn't write "Basic functions are free to use". We don't want to give this impression to the users, fastdup is free. And this is the reason why advances functions we call "for beta testers/early release" and require registration.

Reply:
I've replaced it with "fastdup is free"

Reply:
Let's keep the current bullets unless you have an objection. I like the value prop as is.
 Can you make the same on [visual-layer.com](https://visual-layer.readme.io)?

Reply:
I've reverted the changes to the original bullets. I will keep the readme.io page updated too